spec/unit/loga_spec.rb

```diff
@@ -12,15 +12,17 @@
 
   describe '.configure' do
     it 'configures Loga' do
-      expect(Loga::Configuration).to receive(:new).with(options, {}).and_call_original
+      allow(Loga::Configuration).to receive(:new).and_call_original
```

Comment:
It will be better if you move this one in a `before` or stop the `RSpec/MessageSpies`

Reply:
I think it's ok to keep the setup in the test. I would to start moving away from using too many `before` and `let` to reduce the cognitive load and keep tests local, within reason of course 😉